### PR TITLE
To new ideas to master

### PR DESCRIPTION
**Pull request by jasper-gruhl**

### CODE REVIEW

File: ideas.html [hunk 1]
```diff
@@ -1 +1,15 @@
 
+<!DOCTYPE>
+<html>
+    <head>
+        <meta charset="utf-8">
+    </head>
+    <body>
+        <p>An awesomely fun super cool sandwhich maker game to order sandwhiches.</p>
```
Comment (jgruhl) [inline ideas.html:8]: I like this idea!

**Comment by jasper-gruhl** [comment]
hi place comments here

File: ideas.html [hunk 2]
```diff
@@ -1 +1,15 @@
...
+        <p>A game in wich you try to get a high score. If you do half of coupons for you!</p> 
+        <p>A website for Jon and Prima Deli.</p>
```
Comment (jgruhl) [inline ideas.html:10]: I like this and maybe we can combine a game too!

File: ideas.html [hunk 3]
```diff
@@ -1 +1,15 @@
...
+        <p>A website about things we love.</p>
+        <p>You create your sandwich by draging items into the sandwich anyway you want. Then you place your order and pay.
+  then it asks you if you wnat to play sandwich games. If you get the high score you get 10% of the next sandwich you buy!
```
Comment (lilygruhl) [inline ideas.html:13]: I think these are AWESOME ideas!!!!!

Comment (lilygruhl) [reply ideas.html:13]: 🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗🤗

**Review by jgruhl — APPROVED** [review]

**Pull request merged**